lib/recurly/pager.php

```diff
@@ -45,15 +44,14 @@ public function rewind() {
    */
   public function current()
   {
-    if (empty($this->_count)) {
+    if ($this->count() == 0) {
       return null;
     }
 
     while ($this->_position >= sizeof($this->_objects)) {
       if (isset($this->_links['next'])) {
-        $num_objects = sizeof($this->_objects);
         $this->_loadFrom($this->_links['next']);
-        $this->_position -= $num_objects;
```

Comment:
Do you think this was an accident or was there some intention to this? Can you think of a case where you might be resetting the position to non-zero?